src/redux/books/books.js

```diff
@@ -0,0 +1,37 @@
+export const ADD_BOOK = 'bookStore/src/redux/book/ADD_BOOL';
+export const REMOVE_BOOK = 'bookStore/src/redux/book/REMOVE_BOOK';
```

Comment:
- [ ] in line 1 there is a small typo, it should be `ADD_BOOK` instead of `ADD_BOOL`